build/dependencies.props

```diff
@@ -75,6 +75,7 @@
     <MicrosoftWebXdtPackageVersion>1.4.0</MicrosoftWebXdtPackageVersion>
     <MicrosoftWin32RegistryPackageVersion>4.5.0-preview1-25914-04</MicrosoftWin32RegistryPackageVersion>
     <MonoAddinsPackageVersion>1.3.7</MonoAddinsPackageVersion>
+    <MonoDevelopSdkPackageVersion>1.0.0</MonoDevelopSdkPackageVersion>
```

Comment:
Given that the package name is `MonoDevelop.Sdk` will this break any of our pascal casing code?

Reply:
No, it should be fine. You can test this locally by running `build.cmd /t:GeneratePropsFiles`. Look in `obj/` for the file it creates.

Reply:
Just curious. Why do we need this?

Reply:
> Just curious. Why do we need this?

This is for VS for mac Razor support.